src/log-store/src/error.rs

```diff
@@ -84,6 +85,90 @@ pub enum Error {
         attempt_index: u64,
         location: Location,
     },
+
+    #[snafu(display(
+        "Failed to build a kafka client, broker endpoints: {:?}",
```

Comment:
Always use `Kafka`. The case is different in the error messages.

Reply:
Changed all `kafka` to `Kafka` in error messages.